AuroraEditor/Base/AppPreferences/UI/ThemePreferences/ThemePreviewIcon.swift

```diff
@@ -88,164 +88,165 @@ struct ThemePreviewIcon: View {
 
     private var codeWindow: some View {
         VStack(alignment: .leading, spacing: 4) {
-            block1
-            block2
-            block3
-            block4
-            block5
+//            block1
+//            block2
+//            block3
+//            block4
+//            block5
```

Comment:
Intentional?

Reply:
yes because I removed a lot of things in AuroraTheme so the icon no longer functions. When the theme editor is revamped I'll find a way to get this to work again.